web/app/styles/app.css

```diff
@@ -200,3 +200,129 @@
 .workflow-panel-cp {
   margin: 10px 10px 10px 10px;
 }
+
+input[type="text"]:disabled {
+  background: #dddddd;
+}
+
+select#soflow {
+  -webkit-appearance: button;
+  -webkit-border-radius: 2px;
+  -webkit-box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.1);
+  -webkit-padding-end: 20px;
+  -webkit-padding-start: 2px;
+  -webkit-user-select: none;
+  background-image: url(http://i62.tinypic.com/15xvbd5.png), -webkit-linear-gradient(#FAFAFA, #F4F4F4 40%, #E5E5E5);
```

Comment:
why external link for down arrow? Not sure if icon is not already used across the application. Please use internal copy url otherwise external are subjective to change without prior notice.